### PR TITLE
Release infrastructure

### PR DESCRIPTION
**Pull request by cboesch-gpsw**

### CODE REVIEW

File: doc/dev/howto/release-process.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Release process
+
+This document describes the steps on how to make a new node.gl release.
```
Comment (mbouron-gpsw) [inline doc/dev/howto/release-process.md:3]: `This document describes the steps required to make a new node.gl release.` ?

File: scripts/make-release [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+fi
+
+# Project version bump
+prv_prj_ver=$(cat VERSION)
+new_prj_ver=${cur_year}.${new_digit}
+echo "New project version: $prv_prj_ver → $new_prj_ver"
+
+# libnodegl version bump
+cur_lib_ver=$(cat libnodegl/VERSION)
+echo -n "New libnodegl version [current: $cur_lib_ver]: "
```
Comment (mbouron-gpsw) [inline scripts/make-release:35]: Maybe `Specify|Enter new libnodegl version [current:  $cur_lib_ver]: " to hint the user that he's supposed to enter the version.

File: doc/dev/howto/release-process.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+1. Make sure the [Changelog](/CHANGELOG.md) is not missing any entry
+2. Run `./scripts/make-release` with no argument from the root directory in a
+   clean and up-to-date git state
+3. Indicate the new `libnodegl` version as prompted, following the semantic
+   versioning convention (the version can remain identical)
+4. Check the last commit and tag
+5. Run `git push -n && git push -n --tags`
+6. If everything looks sane, run `git push && git push --tags`
```
Comment (mbouron-gpsw) [inline doc/dev/howto/release-process.md:12]: If everything looks sane and the release commit has been approved by another developer, run [...].

**Comment by mbouron-gpsw** [comment]
Other than my remarks, LGTM overall.

File: doc/dev/howto/release-process.md (hunk 1 above)
Comment (cboesch-gpsw) [reply doc/dev/howto/release-process.md]: Yeah that's a better wording, thanks, fixed.

File: scripts/make-release (hunk 2 above)
Comment (cboesch-gpsw) [reply scripts/make-release]: OK, changed.

File: doc/dev/howto/release-process.md (hunk 3 above)
Comment (cboesch-gpsw) [reply doc/dev/howto/release-process.md]: Good point, changed :D

**Review by mbouron-gpsw — APPROVED** [review]

**Pull request merged**